man/networkctl.xml

```diff
@@ -122,7 +122,8 @@
               <varlistentry>
                 <term>degraded</term>
                 <listitem>
-                  <para>the link has carrier and addresses valid on the local link configured</para>
+                  <para>the link has carrier and addresses valid on the local link configured,
+                  or for bond master, one of bond slaves is in off, no-carrier, or dormant</para>
```

Comment:
…, one of the bonding slave network interfaces is in off, no-carrier or dormant states.